js/index.js

```diff
@@ -446,11 +502,18 @@ function validate_program_inputs() {
 }
 
 programButton.onclick = async () => {
+    programButton.disabled = true
```

Comment:
use standard practice of using semicolons

Reply:
I have added semicolons too as standard practice.PTAL